lib/oppen/pretty_printer.rb

```diff
@@ -93,15 +112,16 @@ def handle_eof
     #
     # @see Token::Begin
     def handle_begin(token)
-      if @scan_stack.empty?
-        @left_total = @right_total = 1
+      if scan_stack.empty?
+        @left_total = 1
+        @right_total = 1
         @left = @right = 0
```

Comment:
choose between assignation on same line or no (I prefer the first):
```ruby
left_total = right_total = 1
left = right = 0
# Or
left_total = 1
right_total = 1
left = 0
right = 0
```

Reply:
Good catch. I like them sorted.